### DISCUSSION TITLE
Any plans to add Fuzer and SpeedApp (formerly known as HDME)?

### DISCUSSION THREAD

**Discussion by Yankees4life** [autobrr/autobrr#367] [Ideas]
Hello

I'm planning on racing here with a few of these sites. Any plans to add these two trackers?

**Comment by zze0s** [comment]
Hi! Yeah that's doable but I need you to gather all the info. We currently only support indexers with IRC announces.

# Indexer request

-  Do they have irc announce? Yes / No
-  Link to existing autodl-irssi .trackers file: Check wiki, forums and autodl-trackers repo

## Announce examples

Please provide 1-5 different announces. And change release titles for good meassure

- `SOME LONG ANNOUNCE`
- `OTHER ANNOUNCE`

## IRC Network

- Network name: TrackerNetwork
- Network address: irc.network.tld
- Port: 6697 is common for TLS
- SSL/TLS: Yes / No
- NickServ registration required? Check wiki, guides etc: Yes / No
- Is there a required format for the nick? Like `username|bot`? Check wiki, guides etc
- Invite command: Eg. `invitebot announce USERNAME IRCKEY`

## Bonus

Check their categories in the announce an on site

Announce category - Site category name

**Reply by zze0s** [reply]
@Yankees4life no sure if you saw this so I'm pinging you now :)

**Reply by Yankees4life** [reply]
@zze0s not sure if you saw my reply below so I'll ping you too

**Comment by Yankees4life** [comment]
Sorry, I heard. I'm gonna have to look into my autodl chatzilla config (and cheatsheet) for the other info but the tracker files are here
https://github.com/mkgeeky/autodl-trackers

EDIT: Ok...here I go

Network name: Fuzer
Announce channel: #autodl
Network address: irc://irc.fuzer.me/Fuzer
Port: 6697
SSL/TLS: Works with either
NickServ registration required? Check wiki, guides etc: Yes
Is there a required format for the nick? Like username|bot? You need to set register on the server with your email and password and you can set your nickname however you like
Invite command: /msg Nickserv identify announce passwordontracker
Announce example
New Torrent: כלואה - תרגום בצד | Some File 2022 1080p BluRay DD5 1 x264 Category: סרטים HD By: User Size: 0GB Link: http://fuzer.me/attachment.php?attachmentid=number [Some.File.2022.1080p.BluRay.DD5.1.x264]

Network name: Speedapp
Announce channel: #speedapp
Network address: irc.p2p-network.net
Port: 6697
SSL/TLS: Works with either. Similar to HDME but its on a different channel
NickServ registration required? Check wiki, guides etc: Yes
Is there a required format for the nick? Like username|bot? You need to set register on the server with your email and password and you can set your nickname however you like
Invite command: /msg Nickserv identify announce rsspasskey
Announce example
New! (Movies HD-RO) - Some.File.2022.1080p.WEB-DL.DDP5.1.Atmos.H.264.mkv - (0 GB) - https://speedapp.io/browse/someurl
